repository/kafka/operator/templates/server.properties.yaml

```diff
@@ -36,18 +36,45 @@ data:
     # The maximum size of a request that the socket server will accept (protection against OOM)
     socket.request.max.bytes={{ .Params.SOCKET_REQUEST_MAX_BYTES }}
 
+    {{ if eq .Params.TRANSPORT_ENCRYPTION_ENABLED "true" }}
+    ############################# TLS Settings #############################
+    ssl.keystore.location=/home/kafka/tls/kafka.server.keystore.jks
+    ssl.keystore.password=changeit
+    ssl.key.password=changeit
+
+    ssl.truststore.location=/home/kafka/tls/kafka.server.truststore.jks
+    ssl.truststore.password=changeit
+    
+    ssl.enabled.protocols=TLSv1.2
+    
+    {{ if .Params.TRANSPORT_ENCRYPTION_CIPHERS }}
```

Comment:
`TRANSPORT_ENCRYPTION_CIPHERS` will always be present, as it's required and has a default value
better remove this condition as it will always be executed